blocks/reader-revenue-manager/subscribe-with-google/block.json

```diff
@@ -0,0 +1,14 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
+	"name": "google-site-kit/rrm-subscribe-with-google",
+	"version": "n.e.x.t",
+	"title": "Subscribe with Google",
+	"category": "widgets",
+	"icon": "google",
+	"description": "Allow users to subscribe using Reader Revenue Manager to access content behind a paywall.",
+	"textdomain": "google-site-kit",
+	"supports": {
+		"inserter": false
```

Comment:
The `inserter` property controls whether the block can be inserted in the editor. It's defaulted to `false`, to be set to `true` for the post editor.